src/test/java/automationpractice/com/pages/AddressPage.java

```diff
@@ -7,11 +7,24 @@
 public class AddressPage extends PageObject {
 
 
+
     @FindBy(xpath = "//*[contains(@class,'button-medium')and not(@rel)]")
     private WebElementFacade addNewAddressButton;
+    @FindBy(xpath = "//*[contains(@class,'alternate_item')]")
+    private WebElementFacade newListName;
+    @FindBy(xpath = "//*[contains(@class,'last_item')]//*[contains(@class,'icon-remove')]")
+    private WebElementFacade deleteButton;
 
 
     public void clickAddNewAddressButton() {
         addNewAddressButton.click();
     }
+
+    public boolean isNewAddressNameVisible() {
+        return newListName.isVisible();
+    }
+
+    public void clickDeleteButton() {
+        deleteButton.click();
+    }
 }
```

Comment:
isNewAddressNameVisible() - tutaj nie powinno być takiej metody
clickDeleteButton(); - to powinno byc clickDeleteAddressButton(final String addressName);